src/graphql/event/kerberos.rs

```diff
@@ -0,0 +1,143 @@
+use super::{country_code, find_ip_customer, find_ip_network, TriageScore};
+use crate::graphql::{customer::Customer, network::Network};
+use async_graphql::{Context, Object, Result};
+use chrono::{DateTime, Utc};
+use review_database as database;
+
+pub(super) struct BlockListKerberos {
+    inner: database::BlockListKerberos,
```

Comment:
Modify the Kerberos event fields to synchronize with giganto-client 0.13.0

petabi/review-database#140